service/controller/resource/monitoring/verticalpodautoscaler/resource_test.go

```diff
@@ -18,14 +18,10 @@ import (
 	"github.com/giantswarm/prometheus-meta-operator/v2/pkg/unittest"
 )
 
-var update = flag.Bool("update", false, "update the output file")
+var update = flag.Bool("update", false, "update the ouput file")
```

Comment:
I guess the `t` was deleted by accident?
```suggestion
var update = flag.Bool("update", false, "update the output file")
```

Reply:
Well, yes I fixed them all :)